Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -124,7 +124,7 @@ public function getHttpRequest()
      * Returns the HTTP response corresponding to the currently handled request
      *
      * @return ResponseInterface|null
-     * @deprecated Don't depend on this method. The HTTP response only exists after the innermost middleware (dispatch) is done. For that stage use a middleware instead.
+     * @deprecated since 6.0. Don't depend on this method. The HTTP response only exists after the innermost middleware (dispatch) is done. For that stage use a middleware instead.
```

Comment:
"Deprecated since 6.0" - should we add when it should be removed, if not in 7.0?

Reply:
Yes, technically we could already remove it. Given using it is pretty useless as of now, it's maybe even sensible?

Reply:
mh.. What about a helpful exception instead?

Reply:
I removed it from the interface with 3e6c754da4d1e3b054723f4fa41013042c54e7e4 and added a `throw new \BadMethodCallException()` here to make it a bit easier for devs to catch this case

Reply:
Doh.. We rely on that part in `Scripts::initializeExceptionStorage()` and `FileStorage::__construct()` and I think @albe an me already figured that out before.. How should we fix that part?

Reply:
Argh, yes right. I think there is one thing we need to consider:
- an exception logged inside the MVC layer/dispatch will not log a meaningful Response at all, so it should probably not log one at all -> those two places need to handle a `null`
- for exceptions to be logged during the middleware-chain, it could log the Response up to that point, but that means we'd need to adjust the chain with another `onStep()` callback, that updates the `Response` in the `RequestHandler`. I would beg the question though: "Does that make sense/provide helpful information?"

Maybe the whole exception storage logging process should be deferred and be done after the process finishes (`register_shutdown_function`?)? At that point it's more likely that `Response` is actually the one that was returned to the browser

Reply:
I think it's fine to skip the response from the output actually.. I doubt that it contains any useful information if an exception is thrown. @kitsunet what do you think?

Reply:
I removed it with 446259ab76d8a4dab1198f9f4da8ad1919632690 for now.. now waiting for CI and approval/objection :)